### PR TITLE
Fix Category header

### PR DESCRIPTION
**Pull request by kosssi**
The design of the category is different that we are on mobile or desktop, so we return a specific structure for mobile.

before | After
-------|------
capture d ecran 2018-12-11 a 14 18 12 | capture d ecran 2018-12-11 a 14 16 09
capture d ecran 2018-12-11 a 15 20 00 | capture d ecran 2018-12-11 a 15 24 04

### CODE REVIEW

File: src/components/Header/Header.styl [hunk 1]
```diff
@@ -3,7 +3,7 @@
 .HeaderFixed
     position sticky
     position -webkit-sticky
-    z-index 1
+    z-index 2
```
Comment (kosssi) [inline src/components/Header/Header.styl]: z-index 1 is not enough, the graph went over.

File: src/ducks/categories/CategoriesHeader.jsx [hunk 2]
```diff
@@ -1,76 +1,132 @@
...
+    )
+  }
+
+  render() {
+    const {
+      breadcrumbItems,
+      breakpoints: { isMobile }
+    } = this.props
+
+    const accounts = this.displayAccounts()
```
Comment (ptbrowne) [inline src/ducks/categories/CategoriesHeader.jsx]: const accountSwitch = this.renderAccountSwitch() ?

File: src/ducks/categories/CategoriesHeader.jsx [hunk 3]
```diff
@@ -1,76 +1,132 @@
...
+    const incomeToggle = this.displayIncomeToggle()
+    const chart = this.displayChart()
```
Comment (ptbrowne) [inline src/ducks/categories/CategoriesHeader.jsx]: `s/display/render` j aurais tendance à mettre display quand ca affiche vraiment, la ca ne fait que `render`

File: src/ducks/categories/CategoriesPage.jsx [hunk 4]
```diff
@@ -78,16 +77,16 @@ class CategoriesPage extends Component {
     ).reverse()
 
     return (
-      <div className={styles['bnk-cat-page']}>
```
Comment (ptbrowne) [inline src/ducks/categories/CategoriesPage.jsx]: 🎉

File: src/ducks/categories/CategoriesHeader.jsx [hunk 5]
```diff
@@ -1,75 +1,116 @@
...
 
     return (
       <div className={styles.CategoriesHeader}>
         <div>
-          <AccountSwitch small={selectedCategory !== undefined} />
-          {selectedCategory && (
-            <BackButton onClick={previousItem && previousItem.onClick} />
-          )}
+          {accounts}
```
Comment (ptbrowne) [inline src/ducks/categories/CategoriesHeader.jsx]: s/accounts/accountSwitch ?

**Pull request merged**